odev/common/actions.py

```diff
@@ -72,15 +73,15 @@ def _action_name(cls) -> str:
 class IntAction(Action):
     """Converter for command line arguments passed as a string that should be converted to an int."""
 
-    def _transform_one(self, value: str) -> int:
-        return int(value)
+    def _transform_one(self, value: Union[str, int]) -> int:
+        return int(value) if isinstance(value, str) else value
```

Comment:
```suggestion
        return int(value)
```
int(3) will return the same integer 3. Am I missing something that would justify this ?

Reply:
Innatention error left after an unsuccessful test, whoops